app/src/main/java/io/pslab/interfaces/sensorloggers/PowerSourceRecordables.java

```diff
@@ -0,0 +1,16 @@
+package io.pslab.interfaces.sensorloggers;
+
+import io.pslab.models.PowerSourceData;
+import io.realm.RealmResults;
+
+public interface PowerSourceRecordables {
+    PowerSourceData getPowerData(long timeStamp);
+
```

Comment:
Excessive line breaks?

Reply:
There are many other classes wth same problem of excessive line breaks. I think i should make a separate clean up PR. will it be okay?

Reply:
For this file do here

Reply:
Done